### DISCUSSION TITLE
Idea: Video recording

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#1020] [Ideas]
Not sure if this is possible but I would love to see an option for in-game video recording.

**Comment by XK9274** [comment]
https://github.com/XK9274/screencap-toolkit-miyoo

I put together a toolkit for this a while ago. It supports:

Screenrecorder/StreamtoRTMP/TwitchStream aswell as a VNC server.

The MMP can't really keep up with recording and it can really struggle to provide video capture and a decent gaming experience on GBA/PS1 but you're welcome to give it a go on anything below this. You can overclock to provide a better experience all round but that's at your own risk, though the cpu overclocker is provided and can be added into the launch.sh for the method of recording you choose.

If you have any issues let me know

**Comment by TooGeekCreations** [comment]
Hi i did put this in my app folder then started screen recording and the whole screen just freeze. Only option is to shut down miyoo mini plus. The recording is starting because i have an mp4 in my output folder but the mp4 is only showing the applications screen because that is where the screen freezes. I do not want to record games only the menu of the device to show of my theme.

**Comment by XK9274** [comment]
In launch.sh for the screen recorder remove the infopanels


________________________________
